hadar/lib/services/authentication/LogInPage.dart

```diff
@@ -80,117 +80,116 @@ class _LogInPageState extends State<LogInPage> {
                       });
-                      UserCredential userCredential = await FirebaseAuth.instance.signInWithEmailAndPassword(
-                          email: email_control.text,
-                          password: pw_control.text);
-
-                       // if (!FirebaseAuth.instance.currentUser.emailVerified) {
-                       //   await FirebaseAuth.instance.currentUser.sendEmailVerification();
-                       //   await FirebaseAuth.instance.signOut();
-                       //   setState(() {
-                       //     _error = 'נא לאמת את הקישור שנשלח לך במיל';
-                       //     show_spinner = false;
-                       //   });
-                       //   return;
-                       // }
-                      bool is_verfied = await DataBaseService().checkIfVerfied(email_control.text);
-                      if(!is_verfied){
-                        await FirebaseAuth.instance.signOut();
-                        setState(() {
-                          _error = ' החשבון שלך עדיין לא אומת על יד המנהל';
-                          show_spinner = false;
-                        });
-                        return;
-                      }
-
-                      print(FirebaseAuth.instance.currentUser);
+                      return;
+                    }
 
+                    print(FirebaseAuth.instance.currentUser);
+
+
+                    Widget curr_widget = await CurrentUser.init_user();
+                    DataBaseService().add_user_token_to_db();
+                    Navigator.pop(context);
+                    Navigator.push(
+                      context,
+                      MaterialPageRoute(builder: (context) => curr_widget),
+                    );
+                    setState(() {
+                      show_spinner = false;
+                    });
+                  } on FirebaseAuthException catch (e) {
+                    if (e.code == 'user-not-found') {
+                      print('No user found for that email.');
+                      setState(() {
+                        email_control.clear();
+                        pw_control.clear();
+                        _error = 'לא נמצא שם משתמש כזה';
+                        show_spinner = false;
+                      });
 
-                      Widget curr_widget = await CurrentUser.init_user();
-                      DataBaseService().add_user_token_to_db();
-                      Navigator.pop(context);
-                      Navigator.push(
-                        context,
-                        MaterialPageRoute(builder: (context) => curr_widget),
-                      );
+                    } else if (e.code == 'wrong-password') {
+                      print('Wrong password provided for that user.');
                       setState(() {
+                        email_control.clear();
+                        pw_control.clear();
+                        _error = 'הסיסמה אינה מתאימה';
```

Comment:
I think the common phrase is `שם משתמש וסיסמה אינם תואמים`. If there is enough screen space for this to be readable. Otherwise, probably `סיסמה שגויה`. Note that you in general don't want to tell unknown user whether or not such a user exists, since it leaks private information about other users. So you can use the phrase above for both cases.